pkgs/by-name/he/hexxy/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  gitUpdater,
+}: let
+  pname = "hexxy";
+  version = "unstable-2024-06-07";
+in
+  buildGoModule {
+    inherit pname version;
+    src = fetchFromGitHub {
+      owner = "sweetbbak";
+      repo = "hexxy";
+      # upstream does not publish releases, i.e., there are no tags
+      rev = "30e0aa5549bbafeb8204fe34b0d37019f9acc975";
+      hash = "sha256-KBgxZD95UT7i/eYeKLm0LVLliKgK/KiJYXVY9zzwbvk=";
+    };
+
+    vendorHash = "sha256-qkBpSVLWZPRgS9bqOVUWHpyj8z/nheQJON3vJOwPUj4=";
+    ldflags = [
+      "-s"
+      "-w"
+    ];
+
+    passthru.updateScript = gitUpdater {};
```

> Upstream does not tag, therefore we use the latest revision with a git updater.

Does `gitUpdater` work in this case?
```suggestion
    passthru.updateScript = unstableGitUpdater { hardcodeZeroVersion = true; };
```